backend/src/main/java/codezap/member/dto/response/UpdateMemberProfileResponse.java

```diff
@@ -0,0 +1,7 @@
+package codezap.member.dto.response;
+
+public record UpdateMemberProfileResponse(
+        String email,
+        String username
```

Comment:
```suggestion
        @Schema(description = "이메일", example = "code@zap.com")
        String email,
        @Schema(description = "사용자명", example = "zappy")
        String username
```
⚡response에도 schema 설정을 해주세요😁